docs/components/alert/index.html

```diff
@@ -1,36 +1,188 @@
-      <div class="alert-text">This is a sample primary alert text.</div>
-    </div>
-    <div class="alert alert-success margin-vertical-2">
-      <div class="alert-text">This is a sample success alert text.</div>
-    </div>
-    <div class="alert alert-info margin-vertical-2">
-      <div class="alert-text">This is a sample info alert text.</div>
-    </div>
-    <div class="alert alert-warning margin-vertical-2">
-      <div class="alert-text">This is a sample warning alert text.</div>
-    </div>
-    <div class="alert alert-danger margin-vertical-2">
-      <div class="alert-text">This is a sample danger alert text.</div>
-    </div>
-  </div>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Alert Component - Wrixby UI</title>
+    <script
+      src="https://kit.fontawesome.com/303bcab846.js"
+      crossorigin="anonymous"
+    ></script>
+    <link type="text/css" rel="stylesheet" href="/main.css" />
+  </head>
 
-  <iframe
-    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C0%29&t=a11y-dark&wt=none&l=htmlmixed&width=888.75&ds=true&dsyoff=20px&dsblur=79px&wc=true&wa=false&pv=34px&ph=52px&ln=true&fl=1&fm=Hack&fs=15.5px&lh=163%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522alert%2520alert-primary%2520margin-vertical-2%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522alert-text%2522%253EThis%2520is%2520a%2520sample%2520primary%2520alert%2520text.%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522alert%2520alert-success%2520margin-vertical-2%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522alert-text%2522%253EThis%2520is%2520a%2520sample%2520success%2520alert%2520text.%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522alert%2520alert-info%2520margin-vertical-2%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522alert-text%2522%253EThis%2520is%2520a%2520sample%2520info%2520alert%2520text.%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522alert%2520alert-warning%2520margin-vertical-2%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522alert-text%2522%253EThis%2520is%2520a%2520sample%2520warning%2520alert%2520text.%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522alert%2520alert-danger%2520margin-vertical-2%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522alert-text%2522%253EThis%2520is%2520a%2520sample%2520danger%2520alert%2520text.%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
-    class="code-snippets code-snippets-md"
-    sandbox="allow-scripts allow-same-origin"
-  >
-  </iframe>
-</div>
+  <body>
+    <header class="navbar">
+      <nav class="nav">
+        <div class="navbar-brand">
+          <a href="/" class="navbar-logo">
+            <img src="/assets/images/WrixbyUI.png" alt="" />
```

Comment:
Suggestion Please also mention something in alt tags too

Reply:
Yeah! Thanks for pointing